packages/adapter-drizzle/src/lib/sqlite.ts

```diff
@@ -159,7 +159,10 @@ export function SQLiteDrizzleAdapter(
         )
         .get()
 
-      return results?.user ?? null
+      if (!results) {
```

Comment:
This would not work would it?
`results` is a Promise and aren't Promises always truthy?

Reply:
Oh, I see... In this case, to get the same behavior as the original I should do something like:
```
return Promise.resolve(results).then((results) => results.user ?? null )
```
correct? 

I still feel off about something though...
 is there a reason why the adapter implementations for postgresql and mysql are async, but sqlite isn't?

Reply:
@kyujulian The problem is Promise.resolve is returning a Promise.
So i would just use async await it makes it much simpler.

Reply:
Hmm got it, that's actually how I've implemented for a custom adapter in another project.

So do you think it might be a useful / valuable PR? to rewrite / refactor the sqlite implementation to be async, like the other ones (mysql and postgres)